roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,157 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = "logging"
+
+    @classmethod
+    def is_active(cls, task_vars):
+        logging_deployed = get_var(task_vars, "openshift_hosted_logging_deploy", default=True)
```

Comment:
I believe the default is `False`, see [playbooks/adhoc/openshift_hosted_logging_efk.yaml#L16](https://github.com/openshift/openshift-ansible/blob/e432f8a35323e8603f0f58b70d996f7487703ec1/playbooks/adhoc/openshift_hosted_logging_efk.yaml#L16)